index.html

```diff
@@ -4,15 +4,11 @@
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <title>Speedometer 2.1</title>
     <link rel="stylesheet" href="resources/main.css">
-    <script src="resources/main.js" defer></script>
-    <script src="resources/benchmark-runner.js" defer></script>
-    <script src="resources/benchmark-report.js" defer></script>
-    <script src="../resources/statistics.js" defer></script>
-    <script src="resources/tests.js" defer></script>
+    <script src="resources/main.mjs" type="module"></script>
 </head>
 <body>
 <main>
-    <a id="logo-link" href="javascript:showHome()"><img id="logo" src="resources/logo.png"></a>
+    <a id="logo"><img src="resources/logo.png" alt="Speedometer"></a>
```

Comment:
a without href is still not focusable.

Reply:
A thanks for the hint. Seems like I didn't check in Safari (Chrome just accepts all links).
And I realized that this didn't work well in FireFox in the old version.
- changed this to `<button>` as well and cleaned up the CSS a bit to style the button better